fenics_ice/prior.py

```diff
@@ -291,6 +300,23 @@ def inv_action(self, x, y):
         y.set_local(self.tmp1.get_local())
         y.apply("insert")
 
+    def sqrt_action(self,x,y):  # sqrt of inv cov: Gamma -1 Gamma 1/2
+                                #                  L M-1 L L-1 M1/2
+                                #                  L M-1 M1/2
+        M_norm = self.M.norm("linf")
+        self.tmp1, terms = A_root_action(self.M, x, tol=1.0e-16, beta=M_norm, max_terms=100000)
```

Comment:
The tolerance is very small

Reply:
with the lumped PC it is much faster, so likely OK